.github/actions/upstream-test/action.yml

```diff
@@ -40,14 +40,32 @@ runs:
           username: ${{ github.repository_owner }}
           password: ${{ inputs.token }}
 
+      - name: Configure docker image tag
+        shell: bash -l {0}
+        run: |
+          image_base_tag=ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ inputs.ubuntu_version }}-${{ inputs.compiler}}-hdf5_${{ inputs.hdf5_version}}-moab_${{ inputs.moab_version }}
+          if [ ${{ inputs.geant4_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-geant4_${{ inputs.geant4_version }}
+          fi
+          if [ ${{ inputs.double_down_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-double_down_${{ inputs.double_down_version }}
+          fi
+          echo "image_base_tag=${image_base_tag}" >> $GITHUB_ENV
```

Comment:
Please always include the geant and DD state in the image tag.  We then don't need this logic since we can just define it in the tag below as we have done in the past.

Reply:
But there is an issue. We can't use  capital letters in the image tag string. It is invalid for docker url. So, I need to change OFF variable. Any alternative suggestion?

Reply:
I am setting this variable as `no`. `OFF=>no`

Reply:
I see - I forgot about that.  Could we use `off` instead of `no`?  Will CMake allow `off` to be treated like `OFF`?

Reply:
Actually, I think we can't use `off` in CMake for `OFF`. However, we can set any string here as we wish here. There are some conditions that only check these strings. But we can't set empty string in workflow matrix. So, let's go with `off`.